network/config.go

```diff
@@ -18,17 +18,20 @@ type Config struct {
 	CertKeyFile    string   `koanf:"certkeyfile"`
 	TrustStoreFile string   `koanf:"truststorefile"`
 
-	// AdvertHashesInterval specifies how often (in milliseconds) the node should broadcasts its last hashes so
-	// other nodes can compare and synchronize.
+	// AdvertHashesInterval specifies how often (in milliseconds) the node should broadcasts its last hashes,
+	// so other nodes can compare and synchronize.
 	AdvertHashesInterval int
+	// QueryPeerDiagnosticsInterval specifies how often (in milliseconds) the node should query its peers for diagnostic information.
+	QueryPeerDiagnosticsInterval int
 }
 
 // DefaultConfig returns the default NetworkEngine configuration.
 func DefaultConfig() Config {
 	return Config{
-		GrpcAddr:             ":5555",
-		EnableTLS:            true,
-		AdvertHashesInterval: 2000,
+		GrpcAddr:                     ":5555",
```

Comment:
I'd say this function is the const itself. They won't be repeated, they're immutable (since the function returns a new instance every time) and they're named.